Mixpanel/JSONHandler.swift

```diff
@@ -50,6 +50,10 @@ class JSONHandler {
 
     private class func makeObjectSerializable(_ obj: MPObjectToParse) -> MPObjectToParse {
         switch obj {
+        case let obj as Double where !obj.isFinite:
+            Logger.info(message: "enforcing string on non-finite numbers")
+            return String(describing: obj)
+
```

Comment:
Thanks @tonysung ! One small nit suggestion would be as the following, let the exception handled by the default.

           case let obj as Double where obj.isFinite:
            return obj
            
           case is String, is Int, is UInt, is UInt64, is Float, is Bool:
            return obj